cmd/airflow.go

```diff
@@ -944,3 +952,7 @@ func prepareDefaultAirflowImageTag(airflowVersion string, httpClient *airflowver
 	}
 	return defaultImageTag
 }
+
+func isValidTemplate(template string) bool {
+	return template == "etl" || template == "dbt-on-astro" || template == "learning-airflow" || template == "generative-ai"
+}
```

Comment:
nit: a possibly bit cleaner way might be: to declare the template variations as a list of strings variable globally (so that it could be reused in other places if required down the line), and check if the template is part of template variations.

i.e.:
```golang
var templateVariations = []string{"etl", "dbt-on-astro", "learning-airflow", "generative-ai"}

return slices.Contains(templateVariations, template)
```

Reply:
Yeah, this make sense. I've updated the logic for this validation.